ShopAPI/ShopAPI/Controllers/ItemsController.cs

```diff
@@ -120,5 +120,26 @@ public async Task<IActionResult> DeleteProduct(int id)
 
             return NoContent();
         }
+
+        /* TODO(epadams)
+        [HttpPut("Item/UpdateStock")]
+        public async Task<IActionResult> UpdateStock(int id, int quantity)
+        {
+
+            return NoContent();
+        }
+        */
+
+        [HttpPut("Item/ChangePrice")]
+        public async Task<IActionResult> ChangePrice(int id, decimal price)
```

Comment:
price should be in a json object to be consistent with the other endpoints. E.g. `{ "price": 124.21 }`.

Reply:
Done